src/types/sequencing.ts

```diff
@@ -65,8 +65,7 @@ export interface ISequenceAdaptation {
     commandDictionary: AmpcsCommandDictionary | null,
     parameterDictionaries: AmpcsParameterDictionary[],
   ) => (context: CompletionContext) => CompletionResult | null;
-  autoIndent: () => (context: IndentContext, pos: number) => number | null | undefined;
-  conditionalKeywords: { else: string; elseIf: string[]; endIf: string; if: string[] };
+  autoIndent?: () => (context: IndentContext, pos: number) => number | null | undefined;
```

Comment:
Are we sure that no one has implemented `conditionalKeywords` in their adaptations that we don't need to add a way to handle deprecated functionality?

Reply:
There is overlapping functionality in core and adaptation and core and I believe this is now redundant, but I'll check with Clipper and confirm.

Reply:
Assuming we get the go ahead that this doesn't break things, we should still follow up and allow missions to customize. I've created this ticket.

https://github.com/NASA-AMMOS/aerie-ui/issues/1517

Reply:
Carter has DM'ed concurrence, we'll schedule 1517 soon